### PR TITLE
bring n-notification into the monorepo

### PR DESCRIPTION
**Pull request by chee**
This component hasn't had a user-facing change since February 2020, and Customer Products are going to be switching to npm-only components over the next months so let's migrate `n-notification` to a "origami v2" component and bring it in

### CODE REVIEW

File: components/n-notification/.circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  test:
+    docker:
+      - image: 'circleci/node:10-browsers'
+    steps:
+      - checkout
+      - run: npm install --only=dev
+      - run: npx origami-ci branch
+  publish_to_npm:
+    docker:
+      - image: 'circleci/node:10'
```
Comment (Lily2point0) [inline components/n-notification/.circleci/config.yml:12]: This might cause issues

Comment (chee) [reply components/n-notification/.circleci/config.yml:12]: woops! good catch, thank you. we'll just get rid of that circle config altogether. this house is a github actions house.

File: .github/workflows/percy-components-n-notification.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      with:
+        cache: 'npm'
+        node-version: 16
+    - run: npm ci
+    - uses: ./actions/percy/
+      with:
+        # To comment as the Origami Fox mascot GitHub user
+        github-token: ${{secrets.ORIGAMI_FOX_ACCESS_TOKEN}}
+      env:
+        PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_COMPONENTS_N_NOTIFICATION }}
```
Comment (JakeChampion) [inline .github/workflows/percy-components-n-notification.yml:46]: note: n-notification currently does not have a project within Precy, this workflow will likely fail until we add the project

Comment (chee) [reply .github/workflows/percy-components-n-notification.yml:46]: https://percy.io/27a07468/n-notification/

File: components/n-notification/MIGRATION.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+Support for Bower and version 2 of the Origami Build Service have been removed.
+Follow [the migration guide on the Origami website](https://origami.ft.com/docs/tutorials/bower-to-npm/).
+
+In addition the following `oLayers` events have been replaced, find and replace them in your project:
+
+- The `oLayers.new` event is removed, use `oOverlay.layerOpen` instead.
+- The `oLayers.close` event is removed, use `oOverlay.layerClose` instead.
+
+_Note: When replacing `o-layers` events (`oLayers.close`, `oLayers.new`) make sure they are used only for responding to `o-overlay` and not other packages your project depends on, which may also fire `o-layers` events. That's unlikely. `o-layers` is deprecated and is not used by any other maintained Origami components._
```
Comment (JakeChampion) [inline components/n-notification/MIGRATION.md:14]: I think this part can be deleted, I can't find a reference to oOverlay or oLayers in n-notification

File: .github/workflows/percy-components-n-notification.yml (hunk 2 above)
Comment (JakeChampion) [reply .github/workflows/percy-components-n-notification.yml:46]: Nice :D

File: components/n-notification/MIGRATION.md (hunk 3 above)
Comment (chee) [reply components/n-notification/MIGRATION.md:14]: woops, yep that's an accident

File: components/n-notification/package.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "n-notification",
```
[the remark this answers is not in the document]
Comment (chee) [reply components/n-notification/package.json]: woo thanks

**Review by JakeChampion — APPROVED** [review]

**Comment by chee** [comment]
yessssss, come on come on

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**